homework/06LessonSix.js

```diff
@@ -13,21 +13,28 @@ var assert = require('assert');
 // in two dimensional space.
 var point = {
 };
-
+  point['x'] = 1;
+  point['y'] = 2;
 // Problem 2:
 // Fill in this Point constructor function so that it defines
 // a class of points with properties x and y.
 // In order to do this, you should add parameters x and y
 // as properties to "this".
 // example: this.propName = propValue;
 function Point(x, y) {
-}
+  this.x = 4;
+  this.y = 4;
+};
 
 // Problem 3:
 // Create a new point using the class constructor Point.
 // Set x to 5, and y to -3
 // This can be done as follows: new Point(someX, someY)
-var anotherPoint;
+var anotherPoint = new Point();
```

Comment:
This also works :D but remember that the constructor `Point(x, y)` takes parameters! We can just provide parameters and x & y will get set automatically. (instead of doing it yourself on lines 34-5)